roles/container_runtime/tasks/package_crio.yml

```diff
@@ -70,10 +70,17 @@
     dest: /etc/sysconfig/crio-network
     src: crio-network.j2
 
+- name: Check for /var/lib/containers/overlay2
```

Comment:
When will this be present or not present, that's what's not clear?  I don't like having this here if it's not useful in all cases.

I would like to hear from whoever decided we needed this in the first place.